### PR TITLE
Fix some Atomic Project OS issue, example: no ceph user exists

### PR DESCRIPTION
**Pull request by harobed**
Fix some Atomic Project OS issue, example: no ceph user exists

### CODE REVIEW

**Comment by leseb** [comment]
I just don't understand the purpose of the patch, where is the issue with the problem description? Please improve your commit message.

**Comment by harobed** [comment]
> I just don't understand the purpose of the patch

I added some comment directly in files change.

File: roles/ceph-common/tasks/main.yml [hunk 1]
```diff
@@ -38,7 +47,8 @@
 - include: ./installs/install_on_redhat.yml
   when:
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: please use:

```
when:
  - foo
  - bar
```

File: roles/ceph-common/tasks/main.yml [hunk 2]
```diff
@@ -91,37 +101,50 @@
     dir_owner: ceph
     dir_group: ceph
     dir_mode: "0755"
-  when: ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
+  when: (ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

```
when:
  - ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
  - not is_atomic
```

File: roles/ceph-common/tasks/main.yml [hunk 3]
```diff
@@ -91,37 +101,50 @@
...
 
 - set_fact:
     dir_owner: root
     dir_group: root
     dir_mode: "0755"
-  when: ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
+  when: (ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

File: roles/ceph-common/tasks/main.yml [hunk 4]
```diff
@@ -91,37 +101,50 @@
...
 
 - set_fact:
     key_owner: root
     key_group: root
     key_mode: "0600"
-  when: ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
+  when: (ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

File: roles/ceph-common/tasks/main.yml [hunk 5]
```diff
@@ -91,37 +101,50 @@
...
 
 - set_fact:
     key_owner: ceph
     key_group: ceph
     key_mode: "0600"
-  when: ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
+  when: (ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

File: roles/ceph-common/tasks/main.yml [hunk 6]
```diff
@@ -91,37 +101,50 @@
...
 
 - set_fact:
     activate_file_owner: ceph
     activate_file_group: ceph
     activate_file_mode: "0644"
-  when: ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer
+  when: (ceph_release_num.{{ ceph_release }} > ceph_release_num.hammer) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

File: roles/ceph-common/tasks/main.yml [hunk 7]
```diff
@@ -91,37 +101,50 @@
...
 
 - set_fact:
     activate_file_owner: root
     activate_file_group: root
     activate_file_mode: "0644"
-  when: ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
+  when: (ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis) and not is_atomic
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: ditto

**Comment by leseb** [comment]
@harobed I don't see any explanation anywhere, the commit message is still empty.

**Comment by ktdreyer** [comment]
Why does Project Atomic not include the `ceph` UID? it should be standard, see https://fedorahosted.org/fpc/ticket/524

**Comment by harobed** [comment]
> Why does Project Atomic not include the ceph UID? it should be standard, see https://fedorahosted.org/fpc/ticket/524

I don't know, I have only this users:

```

**Comment by harobed** [comment]
I think we need to fix `ceph-ansible` before fix Atomic configuration. Isn't it?

**Comment by ktdreyer** [comment]
I think it makes sense to have Project Atomic hard-code the ceph UID, as RHEL and Fedora have done in the `setup` package.

**Comment by leseb** [comment]
Closing this due to non-activity, feel free to re-open.

**Pull request closed without merging**